bin/ci/setup_vmdb_configs.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./bin/ci/setup_vmdb_configs.rb
```

Comment:
Not for this PR, but I never understood why we have a shell file that runs a ruby file.